blazor/dropdown-list/popup-setting.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title: Popup Setting in Blazor DropDown List Component | Syncfusion
+description: Checkout and learn here all about Popup Setting in Syncfusion Blazor DropDown List component and much more.
+platform: Blazor
+control: DropDown List
+documentation: ug
+---
+
+# Popup Setting in Dropdown List
+
+## Change the popup width
+
+You can customize the width of the popup using [PopupWidth](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_PopupWidth). Default value of `PopupWidth` is `100%`.
+
+In the below sample PopupWidth is set as 300px
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/popup-setting/popup-width.razor %}
+
+{% endhighlight %}
+
+![Grouping in Blazor DropdownList](./images/popup-setting/blazor_dropdown_popup-width.png)
+
+## Handling TextOverflow 
+
+When the width of popup is less than text's width, then text-overflow:ellipsis style will be automatically appeared.
+
+In the below sample PopupWidth is set as 100px then text-overflow:ellipsis is automatically applied

```

Comment:
use this content - In the below sample `PopupWidth` is set as `100px` then `text-overflow:ellipsis` is automatically applied.